### DISCUSSION TITLE
Arly Mejia, You can ask "What is TEA Project mean idea" here. This is the right place to ask questions

### DISCUSSION THREAD

**Discussion by kevingzhang** [tearust/teaproject#4] [Q&A] [answered]
can you ask here again?

**Comment by geopor** [comment, marked as answer]
For those looking to understand the main idea behind the TEA Project, this is a concise two minute intro: https://www.youtube.com/watch?v=DSmnP_kgcig

We also have [posts on the teaproject.org website](https://teaproject.org/#/doc_list/What_is_TEA%3F%2FHow_TEA_Fixes_the_Internet.md) putting the TEA Project into the context of existing cloud computing apps and how we use two blockchains for both security and speed.
